usecases/optimization_paper/tex/macros/tex_macros.tex

```diff
@@ -160,3 +160,79 @@
 
 
 
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Shotcuts used by TUM DDMM -------------------------
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\newcommand{\refeq}[1]{Equation \eqref{#1}}
+\newcommand{\boxeq}[1]{%
+  \[\fbox{%
+      \addtolength{\linewidth}{-2\fboxsep}%
+      \addtolength{\linewidth}{-2\fboxrule}%
+      \begin{minipage}{\linewidth}%
+      \begin{equation}#1\end{equation}%
+      \end{minipage}%
+    }\]%
+}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\newcommand{\ee}{\end{equation}}
+\newcommand{\be}{\begin{equation}}
+\newcommand{\ec}{\end{center}}
+\newcommand{\bc}{\begin{center}}
+\newcommand{\eea}{\end{eqnarray}}
+\newcommand{\bea}{\begin{eqnarray}}
+\newcommand{\bd}{\begin{description}}
+\newcommand{\ed}{\end{description}}
+\newcommand{\bi}{\begin{itemize}}
+\newcommand{\ei}{\end{itemize}}
+\newcommand{\pa}{\partial}
+
+\newcommand{\bx}{\bs{x}}
+\newcommand{\bxi}{\bx^{(i)}}
+\newcommand{\bxn}{\bx^{(n)}}
+
+\newcommand{\by}{\bs{y}}
+\newcommand{\byn}{\by^{(n)}}
+\newcommand{\bs}{\boldsymbol}
+
+%\newcommand{\br}{\bs{r}} 
+\newcommand{\brn}{\bs{r}^{(n)}} 
+
+\newcommand{\bz}{\bs{z}}
+\newcommand{\pt}{p^{trans}}
+
+\newcommand{\bxx}{\bs{X}}
+\newcommand{\bxb}{\bar{\bs{X}}}
+\newcommand{\byy}{\bs{Y}}
+
+\newcommand{\bt}{\bs{\theta}}
+\newcommand{\btt}{\bs{\Theta}}
+
+
+\newcommand{\bp}{\bs{\phi}}
+
+\newcommand{\beps}{\bs{\epsilon}}
+\newcommand{\qe}{q_{\epsilon}(\bs{\epsilon})}
+
+\newcommand{\bax}{\bs{A}_x}
+\newcommand{\bbx}{\bs{b}_x}
+\newcommand{\bvx}{\bs{V}_x}
+\newcommand{\bcx}{\bs{C}_x}
+\newcommand{\bdx}{\bs{D}_x}
+\newcommand{\E}{\mathbb{E}}
+
+
+
+\newcommand{\psk}[1]{{\color{red} PSK: #1}}
+\newcommand{\atul}[1]{{\color{green} AA: #1}}
+
+% preamble stuff
+\usepackage{subfigure}
+\usepackage{amsmath,bm}
+\usepackage{amssymb}
+\usepackage{mdframed}
+\usepackage{tikz}
+\usetikzlibrary{graphs,graphs.standard}
```

Comment:
I would not include this in this file.
either create a new file for packages, or directly add it to the main file.
I would suggest to not only create two bib files, as you suggested, but also two of these macro files. 
I will make new issue once this is merged and take care of it.

Reply:
However you seem fit Erik. After merging you can maybe create a separate macro for the TUM. I dont mind that

Reply:
Please move this to a suitable location.

Reply:
Done